CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### citus v11.0.3 (July 5, 2022) ###
+
+* Fixes a bug that prevents using `COMPRESSION` and `CONSTRAINT` on a column
```

Comment:
for https://github.com/citusdata/citus/pull/6023, we could add:

Fixes a bug that prevents adding local tables with materialized views to Citus metadata